src/libraries/Common/src/System/Collections/Generic/SparseArrayBuilder.cs

```diff
@@ -190,7 +190,7 @@ public void Reserve(int count)
         public bool ReserveOrAdd(IEnumerable<T> items)
         {
             int itemCount;
-            if (EnumerableHelpers.TryGetCount(items, out itemCount))
+            if (System.Linq.Enumerable.TryGetNonEnumeratedCount(items, out itemCount))
```

Comment:
Nit: using System.Linq;

Reply:
I wanted to avoid bringing all the enumerable methods into scope since this is a System.Collections namespace.